partials/information-banner.php

```diff
@@ -39,3 +39,38 @@
         <?php
     }
 }
+
+// Page-specific banner
+$enable_banner_on_single_view = hale_get_acf_field_status('enable_banner_on_single_view');
+$show_banner = get_field('show_post_banner');
+
+if ($enable_banner_on_single_view && $show_banner) {
+    $banner_content = "";
+    // Main text
+    $banner_text = get_field('post_banner_text');
+    if ($banner_text && $banner_text != "") {
+        $banner_content .= "<p class='govuk-body page-banner__text'>$banner_text</p>";
```

Comment:
As this is banner text do we need to escape i.e, esc_html( $banner_text) ?

Reply:
Done